google-cloud-datastore/google-cloud-datastore.gemspec

```diff
@@ -19,10 +19,8 @@ Gem::Specification.new do |gem|
   gem.required_ruby_version = ">= 2.0.0"
 
   gem.add_dependency "google-cloud-core", "~> 0.21.0"
-  gem.add_dependency "grpc", "~> 1.0"
-  gem.add_dependency "google-protobuf", "~> 3.0"
-  gem.add_dependency "googleapis-common-protos", "~> 1.3"
-  gem.add_dependency "google-gax", "~> 0.6.0"
+  gem.add_dependency "google-gax", "~> 0.7.1"
+  gem.add_dependency "googleapis-common-protos", "~> 1.3.5"
```

Comment:
Should these be `"~> 1.3", ">= 1.3.5"`? As is, we're restricting the dependency more than it was before. (Not sure what our policy currently is with our dependencies.)